File: 0056-merge-intervals/0056-merge-intervals.cpp
class Solution {
public:
    vector<vector<int>> merge(vector<vector<int>>& l) 
    {
        vector<vector<int>>v;
        sort(l.begin(),l.end());
        int i,j;
        j=0;
        v.push_back(l[0]);
        for(i=0;i<l.size();i++)
        {
            if(l[i][0]<=v[j][1])
            {
                v[j][0]=min(l[i][0],v[j][0]);
                v[j][1]=max(l[i][1],v[j][1]);
            }
            else
            {
                v.push_back(l[i]);
                j+=1;
            }
        }
        return(v);
    }
};

File: 0056-merge-intervals/0056-merge-intervals.py
class Solution:
    def merge(self, l: List[List[int]]) -> List[List[int]]:
        l.sort()
        print(l)
        k=[]
        k.append(l[0])
        j=0
        for i in range(1,len(l)):
            if(l[i][0]<=k[j][1]):
                k[j][1]=max(l[i][1],k[j][1])
                k[j][0]=min(k[j][0],l[i][0])
            else:
                k.append(l[i])
                j+=1
        return(k)
        